lib/chai/core/assertions.js

```diff
@@ -215,6 +215,16 @@ module.exports = function (chai, _) {
   Assertion.addChainableMethod('contains', include, includeChainingBehavior);
   Assertion.addChainableMethod('includes', include, includeChainingBehavior);
 
+  // TODO: Add jsdoc
+
+  function assertFail (msg) {
+    if (msg) flag(this, 'message', msg);
+    var obj = flag(this, 'object');
+    throw new chai.AssertionError(msg, {}, obj);
+  }
+
+  Assertion.addMethod('fail', assertFail);
+
```

Comment:
Curious why you added it as a method for all assertions, rather than just tacking it onto `expect`. I was thinking you would just add `expect.fail = function` much like you did with `should.fail`